src/app/dashboard-widgets/edit-space-description-widget/edit-space-description-widget.component.html

```diff
@@ -3,6 +3,11 @@
     <h2 class="card-pf-title">{{space.attributes.name}}</h2>
   </div>
   <div class="card-pf-body">
-    <div almEditable [editable]="isEditable() | async" (onUpdate)="saveDescription($event)">{{space.attributes.description}}</div>
+    <p>
+      <b>Description</b>
```

Comment:
Edit symbol and title show on a description you *can't* edit, which I don't think is correct.